javascript/ql/lib/semmlecode.javascript.dbscheme

```diff
@@ -1,103 +1,75 @@
 /*** Standard fragments ***/
 
-/** Files and folders **/
-
-@location = @location_default;
-
-locations_default(unique int id: @location_default,
-          int file: @file ref,
-          int beginLine: int ref,
-          int beginColumn: int ref,
-          int endLine: int ref,
-          int endColumn: int ref
-         );
-
-@sourceline = @locatable;
-
-numlines(int element_id: @sourceline ref,
-         int num_lines: int ref,
-         int num_code: int ref,
-         int num_comment: int ref
-        );
-
-files(unique int id: @file,
-      varchar(900) name: string ref);
-
-folders(unique int id: @folder,
-        varchar(900) name: string ref);
-
-
-@container = @folder | @file ;
+/*- Files and folders -*/
 
+/**
+ * The location of an element that is not an expression or a statement.
```

Comment:
In JS this contains table the location of statements and expressions as well. Is that not the case in other languages?